src/index.scss

```diff
@@ -452,3 +452,28 @@ $popup-delay: .3s;
     color: $color-primary !important;
     text-decoration: none !important;
 }
+
+.glimpse-logs {
+    svg {
+        height: 12px !important;
+        width: 12px !important;
+        fill: rgba(white, 0.5) !important;
```

Comment:
Just heads up, if we aim to support IE8, `rgba` colors and `SVG` is not supported in IE8.